### PR TITLE
Load glb models

### PR DESCRIPTION
**Pull request by vanous**
This allows loading glTF (glbs are [specified](https://github.com/mvrdevelopment/spec/blob/main/gdtf-spec.md#regarding-gltf-files)) based models.
Also, it fixes a crash when loading DMX Macros.

### CODE REVIEW

**Comment by vanous** [comment]
If you like to upstream my other additions, let me know. I fixed the color mixing and added CMY and Zoom support. I can make a PR from my WIP branch https://github.com/vanous/blenderDMX/tree/Development .

**Comment by hugoaboud** [comment]
> If you like to upstream my other additions, let me know. I fixed the color mixing and added CMY and Zoom support. I can make a PR from my WIP branch https://github.com/vanous/blenderDMX/tree/Development .

Please do! Sorry for the delay, I'm focused on other projects, but looking for maintainers to help me on this one, so feel free to suggest any kind of change you find necessary, here or via Discord (you can DM me there).

**Review by hugoaboud — APPROVED** [review]

File: gdtf.py
Comment (hugoaboud) [inline gdtf.py:135]: One thing I've noticed after merging: this method should follow the lowerCamelCase standard, so it should be `loadGlb`.
Could you fix this in a future PR?

**Comment by hugoaboud** [comment]
I'll wait for the new PR you've mentioned before packing and tagging a new release.

**Pull request merged**
